### ISSUE TITLE
show nicknamed pokemon stats

### ISSUE THREAD

**Issue by zakijesk** [alucas/Snorlax#31]
before when you don't toggle it it shows the nicknames now it's the opposite if you don't toggle it it shows the stats

**Comment by alucas** [comment]
Yep, that bug was already spotted. Will fix that in the next version. Thx!
